draft-ietf-ppm-dap.md

```diff
@@ -1782,25 +1804,27 @@ Next, the Helper checks that:
 
 If any of these checks fail, then the Helper MUST abort with error
 `invalidMessage`. Additionally, if any prep step appears out of order relative
-to the previous request, then the Helper MAY abort with error `invalidMessage`.
-(Note that a report may be missing, in which case the Helper should assume the
-Leader rejected it.)
-
-[OPEN ISSUE: Issue 438: It may be useful for the Leader to explicitly signal
-rejection.]
+to the previous request, then the Helper MUST abort with error
```

Comment:
Why do we enforce the same ordering between requests for a given job? This seems unnecessarily strict when the Helper can just match based on the report_id.

Reply:
I think the discussion in https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/issues/217 is relevant.